k8s/xdmod-build/bc-xdmod-dev.yaml

```diff
@@ -8,15 +8,13 @@ spec:
     type: Git
     git:
       uri: https://github.com/CCI-MOC/xdmod-cntr.git
-      ref: XdmodK8sDeployment
     contextDir: /
   strategy:
     type: Docker
     dockerStrategy:
       dockerfilePath: Dockerfile.moc-xdmod.dev
       from:
         kind: ImageStreamTag
-        name: moc-xdmod:latest
```

Comment:
Don't we need an imagestream name here?

Reply:
Is this not a image stream?  As in the following:

        >oc get is
        NAME            IMAGE REPOSITORY                                                       TAGS     UPDATED
        moc-xdmod       image-registry.openshift-image-registry.svc:5000/xdmod/moc-xdmod       latest   6 days ago
        moc-xdmod-dev   image-registry.openshift-image-registry.svc:5000/xdmod/moc-xdmod-dev   latest   5 days ago

Reply:
Right, but in this commit, you're *removing* the `name:` element.

Reply:
Yeah, I was misreading things. Neeeever mind.